src/Controller/Api/SubscriptionController.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Kiener\MolliePayments\Controller\Api;
+
+use Shopware\Core\Framework\Routing\Annotation\RouteScope;
+use Shopware\Core\Framework\Validation\DataBag\RequestDataBag;
+use Shopware\Core\Framework\Context;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Kiener\MolliePayments\Service\Subscription\CancelSubscriptionsService;
+
```

Comment:
Write Route scope always before class name Ex.

```suggestion
/**
* @RouteScope(scopes={"api"})
*/
class SubscriptionController extends AbstractController 
{
}
```